src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/data/VisitDto.kt

```diff
@@ -64,7 +64,7 @@ data class VisitDto(
     visitRestriction = visitEntity.visitRestriction.name,
     visitRestrictionDescription = visitEntity.visitRestriction.description,
     visitRoom = visitEntity.visitRoom,
-    visitorConcerns = visitEntity.visitorConcerns,
+    visitorConcerns = visitEntity.visitorConcerns?.let { it.text },
```

Comment:
If we make the notes a list in the visit then we can decouple the migration data completely from the visit journey, ie one list with multiple optional notes - this would be primarily be used by the nomis migration and the ui can ignore it.

Reply:
Yes sure was going to resolve this issue in the other tickets because at this moment we don't have any other types.

Reply:
Consider a dto